diff -ruN build/Src/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e Src_new/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e
--- build/Src/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e	2021-08-05 08:31:29.105524298 -0400
+++ Src_new/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e	2021-08-03 09:46:38.000000000 -0400
@@ -18,11 +18,6 @@
 			{NONE} all
 		end
 
-	SHARED_ES_CLOUD_SERVICE
-		export
-			{NONE} all
-		end
-
 	EV_SHARED_APPLICATION
 		export
 			{NONE} all
@@ -244,7 +239,7 @@
 
 	display_launching_page
 		do
-			if preferences.dialog_data.show_update_manager_dialog then
+			if False then
 				display_version_updater_check (agent display_startup_page)
 			else
 				display_startup_page
diff -ruN build/Src/Src/Eiffel/interface/new_graphical/startup/es_startup_page.e Src_new/Src/Eiffel/interface/new_graphical/startup/es_startup_page.e
--- build/Src/Src/Eiffel/interface/new_graphical/startup/es_startup_page.e	2021-08-05 08:31:30.201747793 -0400
+++ Src_new/Src/Eiffel/interface/new_graphical/startup/es_startup_page.e	2021-08-03 09:51:14.000000000 -0400
@@ -16,8 +16,6 @@
 
 	ES_SHARED_FONTS_AND_COLORS
 
-	SHARED_ES_CLOUD_SERVICE
-
 create
 	make
 
@@ -54,21 +52,6 @@
 				if not license_accepted then
 					switch_to_community_agreement_page
 					show_modal_to_window (win)
-				elseif is_cloud_enabled then
-					if is_logged_in then
-						if
-							attached es_cloud_s.service as cld and then
-							attached cld.active_account as acc
-						then
-							cld.on_account_logged_in (acc)
-						end
-						on_next
-					elseif {ES_IDE_SETTINGS}.cloud_required then
-						switch_to_account_page
-						show_modal_to_window (win)
-					else
-						on_next
-					end
 				else
 					on_next
 				end
@@ -235,17 +218,12 @@
 
 	is_cloud_enabled: BOOLEAN
 		do
-			Result := es_cloud_s.service /= Void
+			Result := False
 		end
 
 	is_logged_in: BOOLEAN
 		do
-			if attached es_cloud_s.service as cld then
-				Result := cld.active_account /= Void
-			else
-					-- ES Cloud is not activated!
-				Result := True
-			end
+			Result := True
 		end
 
 feature -- Text
@@ -288,15 +266,7 @@
 			preferences.misc_data.set_license_accepted (True)
 				-- Save preferences right away.
 			preferences.preferences.save_preferences
-			if is_cloud_enabled then
-				if is_logged_in then
-					on_next
-				else
-					switch_to_account_page
-				end
-			else
-				on_next
-			end
+			on_next
 		end
 
 	on_purchase_selected
diff -ruN build/Src/Src/Eiffel/interface/new_graphical/windows/development_window/initialization/eb_development_window_menu_builder.e Src_new/Src/Eiffel/interface/new_graphical/windows/development_window/initialization/eb_development_window_menu_builder.e
--- build/Src/Src/Eiffel/interface/new_graphical/windows/development_window/initialization/eb_development_window_menu_builder.e	2021-08-05 11:07:37.995901523 -0400
+++ Src_new/Src/Eiffel/interface/new_graphical/windows/development_window/initialization/eb_development_window_menu_builder.e	2021-08-05 11:07:55.036112016 -0400
@@ -33,7 +33,7 @@
 			build_refactoring_menu
 			build_window_menu
 			build_help_menu
-			build_cloud_account_menu
+--			build_cloud_account_menu
 				-- Build the menu bar.
 			build_menu_bar
 		end
