diff -urN EiffelStudio22.05/Src/Eiffel/interface/graphical/foundations/tools/es_shell_tools.e Src/Src/Eiffel/interface/graphical/foundations/tools/es_shell_tools.e
--- EiffelStudio22.05/Src/Eiffel/interface/graphical/foundations/tools/es_shell_tools.e	2023-09-07 20:56:33.108377088 -0400
+++ Src/Src/Eiffel/interface/graphical/foundations/tools/es_shell_tools.e	2023-09-09 10:18:46.450472235 -0400
@@ -241,10 +241,6 @@
 			l_tools.put_last ({ES_TESTING_TOOL})
 			l_tools.put_last ({ES_TESTING_RESULTS_TOOL})
 			l_tools.put_last ({ES_WEB_BROWSER_TOOL})
-
-				-- ES Cloud tool
-			l_tools.put_last ({ES_CLOUD_ACCOUNT_TOOL})
-
 				-- Source Control Management tool
 			l_tools.put_last ({ES_SCM_TOOL})
 
diff -urN EiffelStudio22.05/Src/Eiffel/interface/new_graphical/dialogs/eb_about_dialog.e Src/Src/Eiffel/interface/new_graphical/dialogs/eb_about_dialog.e
--- EiffelStudio22.05/Src/Eiffel/interface/new_graphical/dialogs/eb_about_dialog.e	2023-09-07 20:56:34.308366379 -0400
+++ Src/Src/Eiffel/interface/new_graphical/dialogs/eb_about_dialog.e	2023-09-09 10:42:52.068304183 -0400
@@ -78,7 +78,7 @@
 			copyright_label: EV_LABEL
 			registration_label: EV_TEXT
 			info_label: EV_LABEL
-			l_update_check_link, l_license_link: EV_HIGHLIGHT_LINK_LABEL
+			l_license_link: EV_HIGHLIGHT_LINK_LABEL
 			hsep: EV_HORIZONTAL_SEPARATOR
 			ok_button: EV_BUTTON
 			white_cell: EV_CELL
@@ -107,12 +107,6 @@
 			version_label.align_text_left
 			version_label.set_background_color (bg)
 
-			if {ES_GRAPHIC}.is_standard_edition then
-					-- Check for update ...
-				create l_update_check_link.make_with_text ("Check for update (channel: " + preferences.misc_data.update_channel + ")")
-				l_update_check_link.select_actions.extend (agent check_for_update (l_update_check_link))
-			end
-
 			create l_license_link.make_with_text (interface_names.l_read_license_text)
 			l_license_link.select_actions.extend (agent do (create {EB_LICENSE_COMMAND}).execute end)
 
@@ -132,10 +126,6 @@
 			eiffel_text_box.set_padding (Layout_constants.Default_padding_size)
 			eiffel_text_box.extend (version_label)
 			eiffel_text_box.disable_item_expand (version_label)
-			if l_update_check_link /= Void then
-				eiffel_text_box.extend (l_update_check_link)
-				eiffel_text_box.disable_item_expand (l_update_check_link)
-			end
 			eiffel_text_box.extend (copyright_label)
 			eiffel_text_box.disable_item_expand (copyright_label)
 
diff -urN EiffelStudio22.05/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e Src/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e
--- EiffelStudio22.05/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e	2023-09-07 20:56:34.664363200 -0400
+++ Src/Src/Eiffel/interface/new_graphical/main/es_abstract_graphic.e	2023-09-09 10:14:44.950285747 -0400
@@ -253,7 +253,7 @@
 				-- Save EIS storage when project is closed.
 			eiffel_project.manager.close_agents.extend (agent eis_manager.save_storage)
 
-			launch_interface
+			load_interface
 		end
 
 	launch_interface
diff -urN EiffelStudio22.05/Src/Eiffel/interface/new_graphical/windows/development_window/eb_development_window_tools.e Src/Src/Eiffel/interface/new_graphical/windows/development_window/eb_development_window_tools.e
--- EiffelStudio22.05/Src/Eiffel/interface/new_graphical/windows/development_window/eb_development_window_tools.e	2023-09-07 20:56:36.272348821 -0400
+++ Src/Src/Eiffel/interface/new_graphical/windows/development_window/eb_development_window_tools.e	2023-09-09 10:19:36.538662609 -0400
@@ -189,11 +189,6 @@
 		require
 			not_is_recycled: not is_recycled
 		do
-			if attached {ES_CLOUD_ACCOUNT_TOOL} develop_window.shell_tools.tool ({ES_CLOUD_ACCOUNT_TOOL}) as l_tool then
-				Result := l_tool
-			else
-					-- Cloud service is not activated!
-			end
 		end
 
 	scm_tool: detachable ES_SCM_TOOL
